mreg_cli/bacnet.py

```diff
@@ -0,0 +1,105 @@
+from .cli import Flag, cli
+from .exceptions import HostNotFoundWarning
+from .history import history
+from .host import host
+from .log import cli_error, cli_info, cli_warning
+from .util import (
+    clean_hostname,
+    host_info_by_name,
+    _host_info_by_name,
+    resolve_input_name,
```

Comment:
Some of these imports are unused.

Reply:
Fixed now in b1ce84c1afd93f8751371d2e2355d02d6c930b7a